plugins/input/tcp.md

```diff
@@ -170,6 +187,26 @@ TCPSocket.open('127.0.0.1', 5170) do |s|
   s.write('{"k":"v2"}' + "\n")
 end
 ```
+## Tips
+
+### How to Enable TLS Encryption
+
+Since v1.5.0, `in_tcp` support TLS tranport. Here is configuration example.
```

Comment:
in_tcp seems to support TLS from further back so removing `since v1.5.0` is enough.

Reply:
https://github.com/fluent/fluentd-docs-gitbook/pull/50/commits/d3fd3fa5f1fb91d9f942f55dd11fb13ece3a34dc fixed